clouddrift/plotting.py

```diff
@@ -41,12 +41,12 @@ def plot_ragged(
     *args : tuple
         Additional arguments to pass to ``ax.plot``.
     colors : array-like
-        Colors to use for plotting. If colors is the same shape as longitude and latitude,
-        the trajectories are splitted into segments and each segment is colored according
-        to the corresponding color value. If colors is the same shape as rowsize, the
-        trajectories are uniformly colored according to the corresponding color value.
+        Values to map on the current colormap. If ``colors`` is the shape of ``rowsize``,
+        the data points of each row are uniformly colored according to the color value for the row.
```

Comment:
Should we removed "uniformly" here? It is colored as a function of the values that you supply, not necessarily uniform.

Reply:
Maybe we could say:
```
Colors to use for plotting. If ``colors`` has the same shape as ``rowsize``,
each ``ob`` in a ragged array is colored according to the ``row`` it is associated with.
```